src/utils.ts

```diff
@@ -22,7 +22,7 @@ export const separatorRE = /[,[\]{}\n]|\bimport\b/g
 /**                                                           |       |
  *                    destructing   case&ternary    non-call  |  id   |
```

Comment:
Can you update the comment graph as well? Thanks

Reply:
there you go. Is it fine like that or should I write extends?